Sources/Orbit/Components/Icon.swift

```diff
@@ -411,17 +423,19 @@ struct IconPreviews: PreviewProvider {
                 .padding(.top, .xLarge)
 
             (
-                Text("Text", size: .small)
-                + Icon(sfSymbol, size: .small)
+                Text("Text")
+                + Icon(sfSymbol)
                     .iconColor(.blueNormal)
-                + Icon(sfSymbol, size: .small)
+                + Icon(sfSymbol)
                     .baselineOffset(.xxxSmall)
 
-                + Icon(.informationCircle, size: .small)
+                + Icon(.informationCircle)
                     .iconColor(.blueNormal)
-                + Icon(.informationCircle, size: .small)
+                + Icon(.informationCircle)
                     .baselineOffset(.xxxSmall)
             )
+            .textSize(.small)
+            .iconSize(.small)
```

Comment:
Isn't this something you said we don't need on slack?

Reply:
Updated